auto_update_tests.py

```diff
@@ -2,6 +2,8 @@
 
 import os, sys, subprocess, difflib
 
+os.environ['LD_LIBRARY_PATH'] = 'lib' # find our dynamic libraries
```

Comment:
Oh, please do something, anything, other than messing with `LD_LIBRARY_PATH`. Therein lies madness. Also it won't work on Windows.